### PR TITLE
Scoring file structure

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

**Comment by ahemphill** [comment]
@googlebot I signed it!

File: tools/scoring/score/entity/id.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# The score is:
+#  (number of correctly paired devices) - (number of unpaired result devices) total devices
+
+# Again, reporting and virtual device results should be separable
+
+from score.score import Score
+
+
+class EntityId(Score):
```
Comment (berkoben) [inline tools/scoring/score/entity/id.py:30]: nit: Unless we're literally talking about some sort of identifier key, I'd avoid using ID, preferring "Identification" in this case (also applies to some of the other classes)

**Review by berkoben — CHANGES_REQUESTED** [review]
high level stuff:
- Add copyright notices
- run the linter
- try to make the docstrings (that the linter will make you write) as descriptive as possible.  
- revert unchanged files

Blocks of text form the spec are fine for now, but should eventually be re-written into docstring (and probably not verbatim).

File: tools/scoring/cli.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+def _ParseArgs() -> argparse.ArgumentParser:
+
+    parser = argparse.ArgumentParser(description='Score an instance')
+
+    parser.add_argument(
+        '-ont',
+        '--ontology',
+        dest='ontology',
+        required=False,
+        default='/Users/atom/Documents/work/digitalbuildings/ontology/yaml',
```
[the remark this answers is not in the document]
Comment (ahemphill) [reply tools/scoring/cli.py:29]: Yep — punting on path stuff for the time being.

File: tools/scoring/constants/dimension.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+@unique
+class Dimension(Enum):
+    UNIT_MAPPING = auto()
+    STATE_MAPPING = auto()
+    FIELD_SELECTION = auto()
+    FIELD_NAMING = auto()
+    ENTITY = Enum('entity', ['CONNECTION_ID', 'ID' 'POINT_ID' 'TYPE_ID'])
+
+    def __str__(self):
+        return self.value.lower()
```
[the remark this answers is not in the document]
Comment (ahemphill) [reply tools/scoring/constants/dimension.py:14]: Good catch.

File: tools/scoring/score/entity/id.py (hunk 1 above)
Comment (ahemphill) [reply tools/scoring/score/entity/id.py:30]: Good call.

**Comment by charbull** [comment]
Hey, I had to update this file since there was no code yet in it.
Please uncomment those in your PR: https://github.com/google/digitalbuildings/commit/7504f830a2d82ac053ae006ede91865250018c41

**Comment by ahemphill** [comment]
All set.

**Pull request closed without merging**